src/infinity-menu.js

```diff
@@ -53,9 +52,9 @@ export default class InfinityMenu extends React.Component {
 			return true;
 		}
 
-		if (this.state.search.isSearching &&
-			this.state.search.isSearching === nextState.search.isSearching &&
-			this.state.search.searchInput === nextState.search.searchInput) {
+		 if (this.state.search.isSearching
```

Comment:
this statement is only tell us that if search input is same, we don't need to re render